spec/unit/tariff_synchronizer/taric_update_spec.rb

```diff
@@ -43,43 +43,76 @@
     end
   end
 
-  describe '.correct_filename_sequence?' do
-    subject(:taric_update) { described_class }
+  describe '.correct_recent_filename_sequence?' do
+    subject(:update) { described_class }
+
+    context 'when there are updates with an unbroken sequence' do
+      before do
+        create(:taric_update, :missing, example_date: Date.parse('2021-12-04'), sequence_number: 'foo')
```

Comment:
do we need this in the example, maybe it's not necessary

Reply:
Yeah, we do for now, since in live environments we have these updates and they should be filtered out. This is an implicit test of that